guides/centering_accuracy.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: guides
+title: Guides | Centring Accuracy 
+description: This guide teaches you how to centre the accuracy on the playfield.
+---
+
+# Centring Accuracy
+
+This guide will teach you how to centre the accuracy counter on the playfield.
+
+Note that by doing this, the percentage symbol after the accuracy will disappear. This is due to osu! stopping to load the texture and just the hitbox, due to its size.
+
+## Roughly centring accuracy
+This part of the guide will teach you how to roughly centre the accuracy counter - i.e. centring it 'good enough'. To learn how to perfectly centre accuracy, look at the `Perfectly centring accuracy` part of this guide.
+
+1. Open your skin's folder. The folder can be found by navigating to the `Skins\` directory of your osu! installation. By default this will be located at `C:\Users\[User Name]\AppData\Local\osu!\Skins\`. You can locate this by pressing `CTRL + O` in the osu! window to bring up settings, clicking `Open osu! folder`, and navigating to the `Skins\` directory. Alternatively, you can go to settings, navigate to the `SKIN` section, and click `Open current skin folder`.
+2. Find your resolution's width. This will be the first number in your resolution setting (e.g. 1920 is the width of the resolution 1920x1080). You can find this by opening up osu! settings, navigating to the `GRAPHICS` section, and viewing the `Resolution:` setting.
+3. Open your `score-percent.png` (or `score-percent@2x.png` for resolutions with a height of 800 or above) file in an image editing program and use the following formula to find the width which the file will need to be:
+$\frac{width / 2}{0.57}$ 
+4. Change the canvas width of `score-percent.png` (or `score-percent@2x.png`) to the number you got from step 3.
+5. Reload your skin using `CTRL + ALT + SHIFT + S`. Your accuracy will now be displayed in the centre of the playfield.
+
+## Perfectly centring accuracy
+This part of the guide will teach you how to perfectly centre the accuracy counter. 
+1. Same as in the `Roughly centring accuracy` section of this guide.
+2. Same as in the `Roughly centring accuracy` section of this guide.
+3. Find the width of your `score-0.png` (or `score-0@2x.png`) file. All of your score number files should be the same resolution. Also find the width of your `score-dot.png` (or `score-dot@2x.png`)
+4. Find your score number overlap. This is located inside the skin.ini at the `ScoreOverlap:` command under `[Fonts]`. If it is not specified, it will be `-2` by default.
+5. Open your `score-percent.png` (or `score-percent@2x.png` for resolutions with a height of 800 or above) file in an image editing program and use the following formula to find the width which the file will need to be:
+$\frac{(a / 2) - (5b - 4c)/2}{0.57}$ for SD, or $\frac{(a / 2) - (5b - 8c)/2}{0.57}$ for HD

```

Comment:
I haven't gone through this formula yet, please remind me to check it later before merging

Reply:
check it now now now now now